pnet_packet/src/lib.rs

```diff
@@ -212,3 +213,10 @@ pub mod arp;
 pub mod icmp;
 pub mod icmpv6;
 pub mod vlan;
+
+pub mod util;
+
+// Required to make sure that imports from pnet_macros work
+mod pnet_packet {
```

Comment:
Can't we modify `pnet_macros` to get rid of this requirement? It's a bit ugly.

Reply:
I'll read up on the module system - I ported this hack from the previous code, which has been there since pre-1.0. The module system has changed a bit since then, and I have a far better understanding of Rust. Hopefully we can get rid of it.